### PR TITLE
Source File: add retry on SSHException('Error reading SSH protocol banner') - 2

### PR DESCRIPTION
**Pull request by grubberr**
## What
Try to fix this oncall https://github.com/airbytehq/oncall/issues/1954
I found one bug in my prev PR  https://github.com/airbytehq/airbyte/pull/26115

### CODE REVIEW

**Review by artem1205 — APPROVED** [review]

File: airbyte-integrations/connectors/source-file/source_file/client.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def close(self):
 
     def backoff_giveup(self, error):
         # https://github.com/airbytehq/oncall/issues/1954
-        if isinstance(error, SSHException) and str(error) == "Error reading SSH protocol banner":
+        if isinstance(error, SSHException) and str(error).startswith("Error reading SSH protocol banner"):
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-file/source_file/client.py:99]: Should we have a test changing in that case?

**Review by maxi297 — APPROVED** [review]
Conditional approval: Given tests are updated, this change seems to make sense!

File: airbyte-integrations/connectors/source-file/source_file/client.py (hunk 1 above)
Comment (grubberr) [reply airbyte-integrations/connectors/source-file/source_file/client.py:99]: improved

**Pull request merged**